src/runner.py

```diff
@@ -93,6 +93,12 @@ def __init__(
         self.status = runner_status
         self.instance = instance
 
+        # If the proxy setting are set, then add NO_PROXY local variables.
+        if self.config.proxies.get("http") or self.config.proxies.get("https"):
```

Comment:
Is it an issue if we add the no proxy variable even when the proxy is not configured?

Reply:
Makes sense